Revit_Core_Engine/Convert/MEP/FromRevit/Duct.cs

```diff
@@ -44,7 +45,7 @@ public static partial class Convert
         [Input("settings", "Revit adapter settings.")]
         [Input("refObjects", "A collection of objects processed in the current adapter action, stored to avoid processing the same object more than once.")]
         [Output("ducts", "List of BHoM duct objects converted from a Revit duct elements.")]
-        public static List<BH.oM.MEP.System.Duct> DuctFromRevit(this Autodesk.Revit.DB.Mechanical.Duct revitDuct, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
+        public static List<BH.oM.MEP.System.Duct> DuctFromRevit(this Autodesk.Revit.DB.Mechanical.Duct revitDuct, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null, Discipline discipline = Discipline.Physical)
```

Comment:
`Discipline` method parameter should not be added to this method - if you need it, please make this method `private` and then expose it via one-line public converts for physical and MEP disciplines that would call it.